apps/src/templates/sectionAssessments/MultipleChoiceOverviewTable.jsx

```diff
@@ -84,84 +71,68 @@ class MultipleChoiceOverviewTable extends Component {
     });
   };
 
-  getColumns = (sortable) => {
-    let dataColumns = [
-      {
-        property: 'question',
-        header: {
-          label: commonMsg.question(),
-          props: {
-            style: {
-            ...tableLayoutStyles.headerCell,
-          }},
-          transforms: [sortable],
-        },
-        cell: {
-          props: {
-            style: {
-            ...tableLayoutStyles.cell,
-          }}
-        }
-      },
-      {
-        property: 'percentAnsweredOptionOne',
-        header: {
-          label: commonMsg.answerOptionA(),
-          props: {
-            style: {
-            ...tableLayoutStyles.headerCell,
-            width: 90,
-          }},
-        },
-        cell: {
-          format: answerOptionsOneFormatter,
-          props: {
-            style: {
-            ...tableLayoutStyles.cell,
-            width: 90,
-          }}
-        }
+  getNotAnsweredColumn = (index) => (
+    {
+      header: {
+        label: commonMsg.notAnswered(),
+        props: {style: tableLayoutStyles.headerCell},
       },
-      {
-        property: 'percentAnsweredOptionTwo',
-        header: {
-          label: commonMsg.answerOptionB(),
-          props: {
-            style: {
-            ...tableLayoutStyles.headerCell,
-            width: 90,
-          }},
-        },
-        cell: {
-          format: answerOptionsTwoFormatter,
-          props: {
-            style: {
-            ...tableLayoutStyles.cell,
-            width: 90,
-          }}
-        }
+      cell: {
+        props: {style: tableLayoutStyles.cell},
+      }
+    }
+  );
+
+  getAnswerColumns = (index) => (
```

Comment:
This should be `getAnswerColumn`, since you're only returning one column.

Reply:
Fixed.  Thanks